app/src/utils/MapEditControls.tsx

```diff
@@ -106,7 +106,7 @@ const MapEditControls: React.FC<IMapEditControlsProps> = (props) => {
     map.on(eventHandlers.onCreated, onDrawCreate);
 
     onMounted && onMounted(drawRef.current);
-  }, []);
+  }, [context, onDrawCreate, onMountCreate, props]);
```

Comment:
this will break this component, please remove these - they must have been added when you ran
```
npm run lint:fix
```
we cannot add these dependencies here

Reply:
Yeah we discussed this the other day, maybe we can add an ignore to it? Would be ideal if we can still run lint:fix without worrying about this specific case getting broken

Reply:
yeah, is there a way to tweak the lint:fix?